src/components/Stat/StatsList.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import '../Stat/StatList.css'
+
+function StatList({ 

```

Comment:
very nice clean code. The url in your about section does not work but I was able to look at the app through the pull request everything looks great except your last build failed due to I think you deployed your main branch which does not have the new code which caused the build to fail once you either deploy just your dev branch or merge to main and deploy your main branch you should be good.